tasks.py

```diff
@@ -11,6 +11,8 @@
 
 THIS_DIR = Path(__file__).parent
 SOURCE_DIR = THIS_DIR / "src" / "ixdat"
+TESTS_DIR = THIS_DIR / "tests"
+DEV_SCRIPTS_DIR = THIS_DIR / "development_scripts"
```

Comment:
I don't think that develompent_scripts should be looked at by CI tools. It is the one version-controlled place that exists purely for the whimsical convenience of the ixdat developer.

Ah, but I see below you only run black on DEV_SCRIPTS_DIR. I think that's okay. Maybe a comment on this line saying so?

Reply:
Good point and I will add it.